---
layout: paper
title: "A comparative study of ChIP-seq sequencing library preparation methods"
year: 2016
shortref: "Sundaram et al. BMC Genomics 2016 17:816"
nickname: 
journal: "BMC Genomics"
volume: 17
issue: 1
pages: 816
authors: "<strong>Arvind YM Sundaram</strong>, Timothy Hughes, Shea Biondi, Nathalie Bolduc, Sarah K. Bowman, Andrew Camilli, Yap C Chew, Catherine Couture, Andrew Farmer, John P Jerome, David W Lazinski, Andrew McUsic, Xu Peng, Kamran Shazand, Feng Xu, Robert Lyle*, Gregor D Gilfillan*"
image: /assets/images/papers/2016-10-21-sundaram-chipseq.png
pdf: /assets/pdfs/papers/2016-10-21-sundaram-chipseq.pdf
pdflink: https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-016-3135-y?site=bmcgenomics.biomedcentral.com
github: 
pmid: 27769162
pmcid: PMC5073829 
f1000: 
figshare: 
doi: 10.1186/s12864-016-3135-y
category: paper
published: true
tags: []
---
{% include JB/setup %}

# Abstract 

#### Background
ChIP-seq is the primary technique used to investigate genome-wide protein-DNA interactions. As part of this procedure, immunoprecipitated DNA must undergo Òlibrary preparationÓ to enable subsequent high-throughput sequencing. To facilitate the analysis of biopsy samples and rare cell populations, there has been a recent proliferation of methods allowing sequencing library preparation from low-input DNA amounts. However, little information exists on the relative merits, performance, comparability and biases inherent to these procedures. Notably, recently developed single-cell ChIP procedures employing microfluidics must also employ library preparation reagents to allow downstream sequencing.
#### Results
In this study, seven methods designed for low-input DNA/ChIP-seq sample preparation (Accel-NGS® 2S, Bowman-method, HTML-PCR, SeqPlex™, DNA SMART™, TELP and ThruPLEX®) were performed on five replicates of 1 ng and 0.1 ng input H3K4me3 ChIP material, and compared to a “gold standard” reference PCR-free dataset. The performance of each method was examined for the prevalence of unmappable reads, amplification-derived duplicate reads, reproducibility, and for the sensitivity and specificity of peak calling.
#### Conclusions
We identified consistent high performance in a subset of the tested reagents, which should aid researchers in choosing the most appropriate reagents for their studies. Furthermore, we expect this work to drive future advances by identifying and encouraging use of the most promising methods and reagents. The results may also aid judgements on how comparable are existing datasets that have been prepared with different sample library preparation reagents.

