---
layout: paper
title: "Issues with RNA-seq analysis in non-model organisms: A salmonid example"
year: 2017
shortref: "Sundaram et al. Dev Comp Immunol. 2017 75:38-47"
nickname: 
journal: "Developmental & Comparative Immunology"
volume: 75
issue: 1
pages: 38-47
authors: "<strong>Arvind YM Sundaram </strong>, Torstein Tengs, Unni Grimholt*"
image: /assets/images/papers/
pdf: 
pdflink: 
github: 
pmid:28223254
pmcid: 
f1000: 
figshare: 
doi: 10.1016/j.dci.2017.02.006
category: paper
published: true
tags: []
---
{% include JB/setup %}

# Abstract 

High throughput sequencing (HTS) is useful for many purposes as exemplified by the other topics included in this special issue. The purpose of this paper is to look into the unique challenges of using this technology in non-model organisms where resources such as genomes, functional genome annotations or genome complexity provide obstacles not met in model organisms. To describe these challenges, we narrow our scope to RNA sequencing used to study differential gene expression in response to pathogen challenge. As a demonstration species we chose Atlantic salmon, which has a sequenced genome with poor annotation and an added complexity due to many duplicated genes. We find that our RNA-seq analysis pipeline deciphers between duplicates despite high sequence identity. However, annotation issues provide problems in linking differentially expressed genes to pathways. Also, comparing results between approaches and species are complicated due to lack of standardized annotation.