---
layout: paper
title: "Towards population genomics in non-model species with large genomes: a case study of the marine zooplankton <em>Calanus finmarchicus</em>"
year: 2019
shortref: "Choquet et al. Royal Society 2019"
nickname: 
journal: "Royal Society"
volume: 
issue: 
pages: 
authors: "Marvin Choquet*, Irina Smolina, Anusha KS Dhanasiri, Leocadio Blanco-Bercial, Martina Kopp, Alexander Jueterbock, <strong>Arvind YM Sundaram</strong>, Galice Hoarau"
image: /assets/images/papers/2019-02-13-choquet-calanus.png
pdf: /assets/pdfs/papers/2019-02-13-choquet-calanus.pdf
pdflink: https://royalsocietypublishing.org/doi/pdf/10.1098/rsos.180608
github: 
pmid: 
pmcid: 
f1000: 
figshare: 
doi: 10.1098/rsos.180608
category: paper
published: true
tags: [genome reduced-representation, sequence capture enrichment, <em>Calanus</em> spp.]
---
{% include JB/setup %}

# Abstract 

Advances in next-generation sequencing technologies and the development of genome-reduced representation protocols have opened the way to genome-wide population studies in non-model species. However, species with large genomes remain challenging, hampering the development of genomic resources for a number of taxa including marine arthropods. Here, we developed a genome-reduced representation method for the ecologically important marine copepod <em>Calanus finmarchicus</em> (haploid genome size of 6.34 Gbp). We optimized a capture enrichment-based protocol based on 2656 single-copy genes, yielding a total of 154 087 high-quality SNPs in <em>C. finmarchicus</em> including 62 372 in common among the three locations tested. The set of capture probes was also successfully applied to the congeneric <em>C. glacialis</em>. Preliminary analyses of these markers revealed similar levels of genetic diversity between the two <em>Calanus</em> species, while populations of <em>C. glacialis</em> showed stronger genetic structure compared to <em>C. finmarchicus</em>. Using this powerful set of markers, we did not detect any evidence of hybridization between <em>C. finmarchicus</em> and <em>C. glacialis</em>. Finally, we propose a shortened version of our protocol, offering a promising solution for population genomics studies in non-model species with large genomes.
